react-accordion/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,43 @@
+import './Accordion.css';
+import { useState } from 'react';
+
+type Prop = {
+  topics: {
```

Comment:
It would be cleaner to declare a type that is a single Topic. Then you can use it here and below.